### DISCUSSION TITLE
Chart options to show conversion rate (%) over time

### DISCUSSION THREAD

**Discussion by thisconnect** [plausible/analytics#3098] [Anything Else]
Currently it's not possible to change the chart to show conversion rate in percentage over time. Allowing to change the chart to show conversion rate would be useful as one could visualize changes easy regardless of the amount of conversions..

**Comment by metmarkosaric** [comment]
You can now click on the conversion rate in the top graph to view the trend over time!

**Reply by thisconnect** [reply]
@metmarkosaric thank you 👍
